### DISCUSSION TITLE
disable cheer confetti effect on chat

### DISCUSSION THREAD

**Discussion by sbc-an** [night/betterttv#6628] [Ideas]
way too obnoxious, and redundant on multiple scopes
maybe similar option on youtube live chat as well while you're at it?

**Comment by Herrcookiekiller** [comment]
There is an option to disable this now "Reactions", see in the three dots menu up in the top right of chat.
![grafik](https://github.com/user-attachments/assets/ea3b641d-cba4-42cb-aa8e-4d89a750d19e)

**Reply by sbc-an** [reply]
yeah yt does now, ttv still does not

**Comment by sbc-an** [comment]
since they don't seem to be interested, if anyone wants to do it themselves the filter rule is
`www.twitch.tv##.particle-canvas`

**Comment by night** [comment]
Closing as resolved — Twitch has a native Reactions toggle that disables this confetti effect. In chat, open the three-dots menu and toggle off Reactions. No BTTV setting needed.

**Comment by dclstn** [comment]
Reopening this discussion as it is not solved, there is no disable confetti effect for twitch.

**Reply by dclstn** [reply]
<img width="480" height="594" alt="www twitch tv_ml7support (3)" src="https://github.com/user-attachments/assets/59c95db9-20e4-4917-93f1-1bf521fc0701" />

Does disabling Celebrations fix this issue? @sbc-an

**Reply by sbc-an** [reply]
I'm not sure. I thought that option existed before I posted this and affected gift subs rather than cheer. I will say that just sitting there waiting for someone to cheer has me limited on the testing front though.

At this point I'm thinking there might be some confusion between viewer options and streamer options affecting this? Maybe it's why there's so many false positive "solutions" to this?
